src/resource.js

```diff
@@ -5,26 +5,11 @@ import DisplayRaw from './displayRaw';
 import Schema from './schema';
 import { LocationContext } from "./location";
 
-const Resource = ({ result, links }) => {
-  const { setInclude, toggleField, clearFieldSet, setSort } = useContext(LocationContext);
-  const [schemaUrl, setSchemaUrl] = useState('');
-  const [resourceLinks, setResourceLinks] = useState([]);
-
-  const {data = [], included = [] } = result;
-
-  const loadMeta = () => {
-
-    if (links.hasOwnProperty('describedBy')) {
-      const { describedBy, ...additionalLinks } = links;
-
-      setSchemaUrl(describedBy.href);
-      setResourceLinks(additionalLinks);
-    }
-  };
-
-  useEffect(() => {
-    loadMeta();
-  }, [links]);
+const Resource = ({ links }) => {
+  const { document, setInclude, toggleField, clearFieldSet, setSort } = useContext(LocationContext);
+  const { describedBy = null, ...resourceLinks } = links;
+  const schemaUrl = describedBy ? describedBy.href : '';
```

Comment:
Better to just have a variable than a "state", especially since it won't change without new props.